packages/linear-genome-view/src/LinearGenomeView/index.test.ts

```diff
@@ -276,127 +276,164 @@ test('can navToMultiple', () => {
   )
 })
 
-test('can zoom to displayed regions given a selection', () => {
-  const session = Session.create({
-    configuration: {},
+describe.only('Zoom to selected displayed regions', () => {
```

Comment:
should the describe.only be removed?

Reply:
Yep, that was accidentally left over from my testing. Removed. Thanks.